src/reducers/system.js

```diff
@@ -0,0 +1,30 @@
+import { NOT_FOUND } from '../actionTypes/system';
+
+const initialState = {
+  notFound: {
+    status: false,
+    attribute: 'Page',
+  },
+  progressBar: 0,
+  noInternet: false,
+};
+
+/**
+ *
+ * @param {object} state
+ * @param {object} action
+ * @returns {object} new state
+ */
+const login = (state = initialState, { type, payload }) => {
```

Comment:
Can you please change the name of this reducer into something else more relevant here?